README.md

```diff
@@ -16,6 +16,23 @@ papers and bounding boxes for those entities.
 extract entities and their bounding boxes from papers.
 * `ui/`: the user interface for the augmented PDF reader.
 
+## Quick Start
+
+In order to run the reader app locally (pointing to the production api) you only need to do 
+
+```bash
+cd ui
+npm install 
+npm start
+```
+
+However, if you want to run the api locally as well you'll have to:
+- Do the 1 time setup instructions for the api detailed in [api/README.md#one-time-setup](api/README.md#one-time-setup)
```

Comment:
Readers should also do the `npm install` in the API directory before following the instructions in one-time setup

Reply:
yeah that's fair, I was trying to balance how much detail to put in here but you're right that one is worth including

Reply:
Changed this up so the api/README.md covered the one time setup in the right spot and just referenced those instructions as a whole